src/middleware/navigation/update/has.overseas.entity.middleware.ts

```diff
@@ -1,15 +1,15 @@
 import { Request, Response, NextFunction } from 'express';
 
 import { logger } from '../../../utils/logger';
-import { UPDATE_LANDING_URL } from '../../../config';
+import { SECURE_UPDATE_FILTER_URL } from '../../../config';
 import { getApplicationData } from "../../../utils/application.data";
 import { checkOverseasEntityNumberEntered, NavigationErrorMessage } from '../check.condition';
 
 export const hasOverseasEntityNumber = (req: Request, res: Response, next: NextFunction): void => {
   try {
     if (!checkOverseasEntityNumberEntered(getApplicationData(req.session)) ) {
```

Comment:
remove space at the end of the condition `if (!checkOverseasEntityNumberEntered(getApplicationData(req.session)))`